spraycharles.py

```diff
@@ -33,6 +34,8 @@ def args():
     parser.add_argument("-f", "--fireprox", type=str, dest="fireprox", help="the url of the fireprox interface, if you are using fireprox.", required=False)
     parser.add_argument("-d", "--domain", type=str, dest="domain", help="HTTP: Prepend DOMAIN\\ to usernames. SMB: Supply domain for smb connection", required=False)
     parser.add_argument("--analyze", action="store_true", dest="analyze_results", help="Run the results analyzer after each spray interval. False positives are more likely", required=False)
+    parser.add_argument("--jitter", type=int, dest="jitter", help="Jitter time between requests in seconds.", required=False)
+    parser.add_argument("--jitter_min", type=int, dest="jitter_min", help="Minimum time between requests in seconds.", required=False)
```

Comment:
Change flag from `--jitter_min` to `--jitter-min` and update a few references

Reply:
Around line `190` could you also add an info line that shows the jitter settings? Such as `[*] Jitter: 5-10 seconds between every attempt`